networkx/generators/tests/test_internet_as_graphs.py

```diff
@@ -66,8 +66,8 @@ def set_providers(cls, i):
                         cls.providers[i].add(j)
                     elif j != customer:
                         raise ValueError(
-                            "Inconsistent data in the graph\
```

Comment:
i'm guessing this is a bug in the original?

Reply:
Yeah it's a whitespace issue.